.env

```diff
@@ -5,15 +5,18 @@
 # Docker Compose project name
 COMPOSE_PROJECT_NAME=drupal-tyollisyyspalvelut-helfi
 
+# Local hostname
+DRUPAL_HOSTNAME=drupal-tyollisyyspalvelut-helfi.docker.so
```

Comment:
Should these be in .env.example or something?

Reply:
These should be same for everyone and .env doesn't really contain real secrets so I don't see reason for .env.example file